ghost/core/test/unit/server/data/schema/schema.test.js

```diff
@@ -27,7 +27,6 @@ const VALID_KEYS = {
     ],
     string: [
         'maxlength',
-        'maxLength', // this is the result of a bug and should NOT be used, will be deleted soon
```

Comment:
😮😮😮 now I see how it sneaked it with the subscriptions table 🙈

Reply:
Actually, before yesterday we had no test for this at all 😭 I added `maxLength` so I could merge, and this cleans up the usage

But my god it's easy to miss 😅